templating/templating_app/templates/templating_app/now.html

```diff
@@ -0,0 +1,7 @@
+{% extends 'templating_app/index.html' %}
+
+{% block title %}Now{% endblock %}
+
+{% block content %}
+  It is {% now "DATE_FORMAT" %}.
```

Comment:
Heh, I didn't realize that "DATE_FORMAT" was a real thing.  Can you use examples of the other default formats?